NextcloudTalk/CallKitManager.m

```diff
@@ -243,9 +243,14 @@ - (void)getCallInfoForCall:(CallKitCall *)call
             NCRoom *room = [NCRoom roomWithDictionary:roomDict andAccountId:call.accountId];
             [self updateCall:call withDisplayName:room.displayName];
             
-            NSInteger callFlag = [[roomDict objectForKey:@"callFlag"] integerValue];
-            BOOL hasVideo = (callFlag >= CallFlagWithVideo);
-            [self updateCall:call hasVideo:hasVideo];
+            if ([[NCSettingsController sharedInstance] serverHasTalkCapability:kCapabilityCallFlags forAccountId:call.accountId]) {
+                NSInteger callFlag = [[roomDict objectForKey:@"callFlag"] integerValue];
+                if (callFlag == CallFlagDisconnected) {
+                    [self endCallWithUUID:call.uuid];
+                } else if (callFlag >= CallFlagWithVideo) {
```

Comment:
Can we assume, that when `CallFlagWithPhone` is set it always means there is video? Because that would currently apply to this check (video is 4, phone is 8).
Otherwise I think it should be something like `(callFlag & CallFlagWithVideo) != 0` ?

Reply:
You are right ;) Added bitwise comparison here 6408e08f78e9ff5471dd5268103055df9d687dd4